service/src/main/kotlin/fi/espoo/evaka/varda/VardaDecisions.kt

```diff
@@ -33,6 +35,14 @@ fun updateDecisions(h: Handle, client: VardaClient) {
     sendUpdatedDecisions(h, client)
 }
 
+fun removeMarkedDecisions(h: Handle, client: VardaClient) {
+    val decisionIds: List<Long> = getDecisionsToDelete(h)
+    decisionIds.forEach { id ->
+        client.deleteDecision(id)
+        softDeleteDecision(h, id)
```

Comment:
We should check if the delete from varda succeeds before touching evaka data
```suggestion
        if (client.deleteDecision(id)) {
          softDeleteDecision(h, id)
        }
```